src/hashes/constants.rs

```diff
@@ -0,0 +1,136 @@
+//! Constants used in hashes module, such as coefficients of polynomial of degree 129!
+use crate::{algebra::field::prime::AESField, Field};
+
+/// Represents the coefficients of field polynomial used in GCM
+/// f = 1 + α + α^2 + α^7 + α^128
+pub const GCMF_IRREDUCIBLE_POLYNOMIAL_COEFFICIENTS: [AESField; 129] = [
```

Comment:
I wonder if there is a better way to store this

Reply:
I was looking into it for a good hour, though I couldn't find anything better. :(
Maybe I should add a test to sanity check the coefficients.